doc/INSTALL-virtualenv.rst

```diff
@@ -20,23 +20,11 @@ Create a Virtual Environment
 Install the Zope2 2.13.26 Software Packages
 -------------------------------------------
 
-First make sure to have a compatible pip version. pip 8.0 and later
-are not supported.
-
-.. code-block:: sh
-
-   $ bin/pip install -U 'pip<8.0'
-   Installing collected packages: pip
-     Found existing installation: pip 9.0.1
-       Uninstalling pip-9.0.1:
-         Successfully uninstalled pip-9.0.1
-   Successfully installed pip-7.1.2
-
 .. code-block:: sh
 
    $ bin/pip install \
-    --trusted-host download.zope.org \
-    --index http://download.zope.org/Zope2/index/2.13.26/ Zope2
+    --no-binary zc.recipe.egg \
+    -r https://raw.githubusercontent.com/zopefoundation/Zope/gh-pages/releases/2.13.26/requirements.txt
```

Comment:
You could use https://zopefoundation.github.io/Zope/releases/2.13.26/requirements.txt instead

Reply:
Die URL hatte ich gesucht und nicht gefunden :)